src/engine/ExportQueryExecutionTrees.h

```diff
@@ -133,14 +120,15 @@ class ExportQueryExecutionTrees {
    *        then the query result will be obtained via `qet->getResult()`.
    * @return a 2D-Json array corresponding to the IdTable given the arguments
    */
-  static nlohmann::json idTableToQLeverJSONArray(
+  static cppcoro::generator<std::string> idTableToQLeverJSONBindingsStream(
```

Comment:
`idTableToQLeverJSONBindings`  suffices (the ` Stream` is not needed anymore, see below).